fastlane/Fastfile

```diff
@@ -160,15 +160,31 @@ platform :android do
     )
   end
 
-  desc "Build purchases module integration tests pointing to load shedder"
-  lane :build_load_shedder_purchases_integration_tests do |options|
-    build_purchases_integration_tests(
+  desc "Build and run purchases module load shedder integration tests"
+  desc "This requires the google cloud cli to be installed and initialized."
+  lane :run_load_shedder_purchases_integration_tests do |options|
+    begin
+      google_project_id = ENV['GOOGLE_PROJECT_ID'] || UI.user_error!('Google project id not found.')
+      build_purchases_integration_tests(
         app_name: 'loadShedderIntegrationTest',
         api_key: ENV['LOAD_SHEDDER_REVENUECAT_API_KEY'],
         google_purchase_token: ENV['LOAD_SHEDDER_GOOGLE_PURCHASE_TOKEN'],
         product_id_to_purchase: ENV['LOAD_SHEDDER_PRODUCT_ID_TO_PURCHASE'],
         base_plan_id_to_purchase: ENV['LOAD_SHEDDER_BASE_PLAN_ID_TO_PURCHASE']
-    )
+      )
+      sh "gcloud firebase test android run --type instrumentation \
+                        --app ../purchases/test_artifacts/loadShedderIntegrationTest-app.apk \
+                        --test ../purchases/test_artifacts/loadShedderIntegrationTest-test.apk \
+                        --timeout 2m \
+                        --results-bucket cloud-test-#{google_project_id}"
+      sh "mkdir -p ~/gsutil/"
+      sh "gsutil -m cp -r -U `gsutil ls gs://cloud-test-#{google_project_id} | tail -1` ~/gsutil/ | true"
```

Comment:
we should either clean up this line or at the very least comment it thoroughly, so we don't have to ask chatgpt what it does